quantum/rgb_matrix/rgb_matrix.c

```diff
@@ -425,7 +427,10 @@ void rgb_matrix_task(void) {
             break;
         case RENDERING:
             rgb_task_render(effect);
-            if (effect) {
+#ifndef RGBMATRIX_INDICATORS_OVERRIDE_RGB_OFF
+            if (effect)
+#endif
+            {
```

Comment:
I assume this is meant to still call it when using `RGB_MATRIX_NONE`.
Note that using `effect` there was changed in: https://github.com/qmk/qmk_firmware/commit/12568fb5a9167d29a52f79c739f11830bde3e4be (#11308), i.e. previously it would work as with the new `#define`.